vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -172,7 +172,9 @@ components:
       type: object
       properties:
         cpus:
-          $ref: '#/components/schemas/CpuConfig'
+          minimum: 1
+          default: 1
+          type: integer
```

Comment:
I'd prefer if we keep a structure for `CpuConfig`. We may extend it to include more information in the future.